BCStudentSoftwareDevTeam/celts
# Add banned users indicators

Fixes #659.

Added a flasher in kiosk mode that alerts when a banned user attempts to log in. Also added an eligible column in the track volunteers table. Finally, when the user searches someone in the track volunteers modal and clicks on them, a warning icon will display beside their name if they are banned from the program associated with that event.

To test:

Fix 1: Sign in with a banned user in kiosk mode. You should see the flasher. 
Fix 2: You should see an "Eligible" column in the volunteer table.
Fix 3: Select a banned user in the volunteer modal. You should see the icon.

_Note: In the test database, khatts is banned in Adoption 101._
